src/decisionengine/framework/modules/EmptySource.py

```diff
@@ -0,0 +1,32 @@
+"""
+This dummy source takes the name of a source datablock
+from config file as parameter "datablock_type" and produces
+an empty pandas DataFrame as a datablock with that name
+"""
+import pandas as pd
+
+from decisionengine.framework.modules import Source
+from decisionengine.framework.modules.Source import Parameter
+
+
+@Source.supports_config(
+    Parameter("datablock_type", default=""),
+)
+class EmptySource(Source.Source):
+    def __init__(self, config):
+        super().__init__(config)
+        self.logger = self.logger.bind(
+            class_module=__name__.split(".")[-1],
+        )
```

Comment:
This can be removed now that `self.logger` is provided by the `Source` base class.